src/composable/orderTypes/Twap.spec.ts

```diff
@@ -129,26 +179,26 @@ describe('Twap', () => {
       isValid: false,
       reason: 'InvalidSpan',
     })
-    expect(Twap.fromData({ ...TWAP_PARAMS_TEST, appData: constants.AddressZero }).isValid()).toEqual({
-      isValid: false,
-      reason: 'InvalidData',
-    })
   })
 
-  test('isValid: Fails if appData has a wrong number of bytes', () => {
+  test('Invalid twap: InvalidData (ABI parse error in appData)', () => {
     // The isValid below test triggers a throw by trying to ABI parse `appData` as a `bytes32` when
     // it only has 20 bytes (ie. an address)
     expect(Twap.fromData({ ...TWAP_PARAMS_TEST, appData: constants.AddressZero }).isValid()).toEqual({
       isValid: false,
       reason: 'InvalidData',
     })
```

Comment:
This test is identical to https://github.com/cowprotocol/cow-sdk/pull/154/files#diff-4b89c7108b87cda7a7ae6bf0bc217684021f6ea46596743c96cfa0b657205d34R162

Reply:
good catch!